user-service/src/main/java/uk/gov/gchq/palisade/service/user/stream/SerDesConfig.java

```diff
@@ -33,15 +33,15 @@
 import java.nio.charset.Charset;
 
 /**
- * The static configuration for kafka key/value serializers/deserializers
- * - Each input has a pair of key/value deserializers
- * - Each output has a pair of key/value serializers
- * In general, the keys are not used, so the choice of serializer is not important
+ * The static configuration for kafka key/value serialisers/deserialisers
+ * - Each input has a pair of key/value deserialisers
+ * - Each output has a pair of key/value serialisers
+ * In general, the keys are not used, so the choice of serialiser is not important
```

Comment:
this reads badly, don't say they aren't important because they may be in future.

Reply:
This line has been removed.